po/common/common.pot

```diff
@@ -5067,3 +5067,12 @@ msgstr "Missing ingredients list"
 msgctxt "missing_nutrition_facts"
 msgid "Missing nutrition facts"
 msgstr "Missing nutrition facts"
+
+msgctxt "ecoscore_p"
```

Comment:
Is this "p" for plural?
```suggestion
# Plural noun for ecoscore
msgctxt "ecoscore_p"
```

Reply:
It is for plural, but for the Eco-score there's no plural, we don't want Eco-scores.